README.md

```diff
@@ -76,6 +76,7 @@ Please use the following BibTex entry for citing Lattigo:
 
 1. Somewhat Practical Fully Homomorphic Encryption (<https://eprint.iacr.org/2012/144>)
 1. Computing accross Trust Boundaries using Distributed Homomorphic Cryptography (<https://eprint.iacr.org/2019/961>)
+1. Multiparty Homomorphic Encryption: From Theory to Practice (https://eprint.iacr.org/2020/304.pdf)
```

Comment:
This line should replace the previous one (eprint.iacr.org/2019/961).

Reply:
OK - removed